src/sentry/static/sentry/app/components/events/eventAttachments.tsx

```diff
@@ -93,6 +96,49 @@ class EventAttachments extends React.Component<Props, State> {
     }));
   };
 
+  getInlineAttachmentRenderer(attachment: EventAttachment) {
+    const contentType = attachment.headers['Content-Type'] || '';
+    const mimeType = contentType.split(';')[0].trim();
+    switch (mimeType) {
+      case 'text/plain':
+        return attachment.size > 0 ? LogFileViewer : undefined;
```

Comment:
Do we want to add this check before the switch so that we don't have to do it for every case?

Reply:
I was thinking maybe empty attachments do have some meaning in cases. But not sure. I would say the moment someone adds a new mime type we can see :D